### PR TITLE
chore: update Q3D busbar example with Matrix Reduction Operation

### PR DESCRIPTION
**Pull request by tizianrot**
## Description
1 -Added Matrix Reduction Operation as per User's Request
2- Upon Report Creating (line 180), category="ACL" seems not be read, not error given but report in the GUi is empty

## Checklist
Please complete the following checklist before submitting your pull request:
- [x ] I have followed the example template and guide lines to add/update an example.
- [x ] I have tested the example locally and verified that it is working with the latest version of AEDT.
- [ x] I have verified that these changes to the best of my knowledge do not introduce any security vulnerabilities.

### CODE REVIEW

File: examples/high_frequency/emc/busbar.py [hunk 1]
```diff
@@ -117,18 +117,56 @@
 print(q3d.net_sources("Bar2"))
 print(q3d.net_sources("Bar3"))
 
+# Create Matrix Reduction Operations
+
+# Series of Bar1 and Bar2
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:122]: @tizianrot, I guess "Create Matrix Reduction Operations" is the title of this new section so we should follow the correct level for the headers.

```suggestion
# ## Create Matrix Reduction Operations
#
# Series of Bar1 and Bar2
```

File: examples/high_frequency/emc/busbar.py [hunk 2]
```diff
@@ -117,18 +117,56 @@
...
+mr_series = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_1_Series",
+    new_net_name="Series1",
+)
+
+# Add Parallel with Bar3
+
+# mr_series = [b for b in q3d.matrices if b.name == 'MR_1_Series'] [0]
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:133]: @tizianrot I don't think this is needed, right?
```suggestion
```

File: examples/high_frequency/emc/busbar.py [hunk 3]
```diff
@@ -117,18 +117,56 @@
...
+# Series of Bar1 and Bar2
+
+mr_series2 = q3d.insert_reduced_matrix(
+    operation_name="JoinSeries",
+    assignment=["Sink1", "Source2"],
+    reduced_matrix="MR_2_series",
+    new_net_name="Series2",
+)
+
+# Add Series with Bar3
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:151]:
```suggestion
# Add Series with Bar3

```

File: examples/high_frequency/emc/busbar.py [hunk 4]
```diff
@@ -117,18 +117,56 @@
...
 setup1 = q3d.create_setup(props={"AdaptiveFreq": "100MHz"})
 sweep = setup1.add_sweep()
-sweep.props["RangeStart"] = "1MHz"
-sweep.props["RangeEnd"] = "100MHz"
+sweep.props["RangeStart"] = "0MHz"
+sweep.props["RangeEnd"] = "1000MHz"
 sweep.props["RangeStep"] = "5MHz"
 sweep.update()
 
 # ### Set up for postprocessing
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:167]:
```suggestion
# ## Set up for postprocessing
```
@tizianrot be consistent with the other headers.

File: examples/high_frequency/emc/busbar.py [hunk 5]
```diff
@@ -137,9 +175,23 @@
     get_self_terms=False, get_mutual_terms=True, category="C"
 )
 
+# Specify the traces to display after solving the model.ACL - Reduced Matrix MR_1_Series
+
+data_reducedm1_plot_self = q3d.matrices[1].get_sources_for_plot(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:180]: @tizianrot can we find easier to read variable names? I mean for data_reducedm1_plot_self and data_reducedm2_plot_self

**Review by gmalinve — CHANGES_REQUESTED** [review]

**Comment by gmalinve** [comment]
@tizianrot I think that as soon as the post-processing part starts you could add a line explaining how to get available report categories with: q3d.post.available_quantities_categories() because I see that you specify the category in the get_traces_for_plot.

File: examples/high_frequency/emc/busbar.py [hunk 6]
```diff
@@ -117,18 +117,66 @@
...
+
+# Add Series with Bar3
+
+mr_series2.add_operation(
+    operation_type="JoinSeries",
+    source_names=["Sink3", "Source1"],
+    new_net_name="MR_2_Series1",
+)
+
 # Create the solution setup and define the frequency range for the solution.
```
Comment (anur7) [inline examples/high_frequency/emc/busbar.py:159]: @tizianrot maybe here you can create a section for the creation of the setup and analysis differentiating from the previous section "Create Matrix Reduction Operations"

File: examples/high_frequency/emc/busbar.py [hunk 7]
```diff
@@ -117,18 +117,66 @@
...
-sweep.props["RangeStart"] = "1MHz"
-sweep.props["RangeEnd"] = "100MHz"
+freq_sweep_name = "my_sweep"
+setup1 = q3d.create_setup(props={"AdaptiveFreq": "1000MHz"})
+sweep = setup1.add_sweep(name=freq_sweep_name)
+sweep.props["RangeStart"] = "0Hz"
+sweep.props["RangeEnd"] = "20GHz"
 sweep.props["RangeStep"] = "5MHz"
 sweep.update()
+sweep.update()
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:168]:
```suggestion
```

File: examples/high_frequency/emc/busbar.py (hunk 4 above)
Comment (anur7) [reply examples/high_frequency/emc/busbar.py:177]: @tizianrot the title of each code section, in this case "Set up for postprocessing", begins with 3 hashtag, so please apply @gmalinve suggestion

**Review by gmalinve — CHANGES_REQUESTED** [review]

File: examples/high_frequency/emc/busbar.py [hunk 8]
```diff
@@ -137,19 +185,40 @@
     get_self_terms=False, get_mutual_terms=True, category="C"
 )
 
-# Define a plot and a data table in AEDT for visualizing results.
+# Specify the traces to display after solving the model. ACL - Reduced Matrix MR_1_Series
 
-q3d.post.create_report(expressions=data_plot_self)
-q3d.post.create_report(
-    expressions=data_plot_mutual, context="Original", plot_type="Data Table"
+data_red_m1_plot_self = q3d.matrices[1].get_sources_for_plot(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:190]:
```suggestion
data_red_m1_plot_self = mr_series.get_sources_for_plot(
```

File: examples/high_frequency/emc/busbar.py [hunk 9]
```diff
@@ -137,19 +185,40 @@
...
 )
 
-# ## Analyze
-#
-# Solve the setup.
+# Specify the traces to display after solving the model. ACL - Reduced Matrix MR_2_Series
 
-q3d.analyze(cores=NUM_CORES)
-q3d.save_project()
+data_red_m2_plot_self = q3d.matrices[2].get_sources_for_plot(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:196]:
```suggestion
data_red_m2_plot_self = mr_series2.get_sources_for_plot(
```

File: examples/high_frequency/emc/busbar.py [hunk 10]
```diff
@@ -117,18 +117,66 @@
...
+
+q3d.analyze(cores=NUM_CORES)
+q3d.save_project()
 
 # ### Set up for postprocessing
 #
-# Specify the traces to display after solving the model.
+# Specify the traces to display after solving the model. Capacitances - Original Matrix
 
 data_plot_self = q3d.matrices[0].get_sources_for_plot(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:181]:
```suggestion
original_matrix_self = q3d.matrices[0].get_sources_for_plot(
```
and the other one below that I can't comment: data_plot_mutual -> original_matrix_mutual

File: examples/high_frequency/emc/busbar.py [hunk 11]
```diff
@@ -137,19 +185,40 @@
...
 
-q3d.analyze(cores=NUM_CORES)
-q3d.save_project()
+data_red_m2_plot_self = q3d.matrices[2].get_sources_for_plot(
+    get_self_terms=True, get_mutual_terms=False, category="ACL"
+)
+
+# Define plots and a data table in AEDT for visualizing results.
+
+rep_original_self_c = q3d.post.create_report(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:202]:
```suggestion
original_matrix_self_report = q3d.post.create_report(
```

File: examples/high_frequency/emc/busbar.py [hunk 12]
```diff
@@ -137,19 +185,40 @@
...
+    expressions=data_plot_self, plot_name="Original, Self Capacitances"
+)
+rep_original_mutual_c = q3d.post.create_report(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:205]:
```suggestion
original_matrix_mutual_report = q3d.post.create_report(
```

File: examples/high_frequency/emc/busbar.py [hunk 13]
```diff
@@ -137,19 +185,40 @@
...
+    expressions=data_plot_mutual,
+    context="Original",
+    plot_type="Data Table",
+    plot_name="Original, Mutual Capacitances",
+)
+rep_red_m1_self_acl = q3d.post.create_report(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:211]:
```suggestion
reduced_matrix_1_self_report = q3d.post.create_report(
```

File: examples/high_frequency/emc/busbar.py [hunk 14]
```diff
@@ -137,19 +185,40 @@
...
+    expressions=data_red_m1_plot_self,
+    context="MR_1_Series",
+    plot_name="MR_1_Series, Self Inductances",
+)
+rep_red_m2_self_acl = q3d.post.create_report(
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:216]:
```suggestion
reduced_matrix_2_self_report = q3d.post.create_report(
```

File: examples/high_frequency/emc/busbar.py [hunk 15]
```diff
@@ -117,18 +117,66 @@
 print(q3d.net_sources("Bar2"))
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:117]: @tizianrot here and above these lines there are a series of prints, do you think are really needed?

File: examples/high_frequency/emc/busbar.py [hunk 16]
```diff
@@ -117,18 +117,66 @@
...
+sweep.update()
+
+# ## Analyze
+#
+# Solve the setup.
+
+q3d.analyze(cores=NUM_CORES)
+q3d.save_project()
 
 # ### Set up for postprocessing
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:177]:
```suggestion
# ## Set up for postprocessing
```

File: examples/high_frequency/emc/busbar.py [hunk 17]
```diff
@@ -117,18 +117,66 @@
...
 #
-# Specify the traces to display after solving the model.
+# Specify the traces to display after solving the model. Capacitances - Original Matrix
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:179]: @tizianrot I'd put in each section the type of matrices you are going to get.
like #Capacitances - Original Matrix and then the following # ACL - Reduced Matrix MR_1_Series
etc. I think is more clear.

**Review by gmalinve — CHANGES_REQUESTED** [review]

File: examples/high_frequency/emc/busbar.py [hunk 18]
```diff
@@ -117,31 +117,56 @@
...
+# Add Series with Bar3
+
+mr_series2.add_operation(
+    operation_type="JoinSeries",
+    source_names=["Sink3", "Source1"],
+    new_net_name="MR_2_Series1",
+)
+
+# ## Add a solution Setup and an interpolating frequency sweep
+#
```
Comment (gmalinve) [inline examples/high_frequency/emc/busbar.py:160]:
```suggestion
```

**Review by gmalinve — APPROVED** [review]
LGTM

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, finally fixed with main

**Pull request merged**